torchgen/model.py

```diff
@@ -692,16 +696,27 @@ def from_yaml(
             dispatch[DispatchKey.CompositeImplicitAutograd] = BackendMetadata(
                 cpp.name(func), structured=False, cpp_namespace=DEFAULT_KERNEL_NAMESPACE
             )
+            dispatch[
+                DispatchKey.CompositeImplicitAutogradNestedTensor
+            ] = BackendMetadata(
+                cpp.name(func), structured=False, cpp_namespace=DEFAULT_KERNEL_NAMESPACE
+            )
```

Comment:
I think we still want to get rid of this, as per https://github.com/pytorch/pytorch/pull/82754#discussion_r950515109.

In the yaml, this:
```
func: foo(...) -> ...:
```
is syntactic sugar for:
```
func foo(...) -> ...:
  dispatch:
    CompositeImplicitAutograd: foo
```

That shouldn't need any extra changes for nested tensors, because nested tensors always opt into `CompositeImplicitAutograd` (unless they have a kernel registered to `CompositeImplicitAutogradNestedTensor`, which takes higher priority)